variation/main.py

```diff
@@ -45,7 +58,7 @@ class Tags(Enum):
 app = FastAPI(
```

Comment:
I think you can do all of the FastAPI meta/schema definition things in a single function call now, a la https://github.com/cancervariants/gene-normalization/blob/787908c57fbf684317a90869071ccb857c4478e9/gene/main.py#L23